### ISSUE TITLE
Add API endpoints to add / remove validators

### ISSUE THREAD

**Issue by jchen86** [jchen86/staker-notifier#5]

**Comment by yury-dubinin** [comment]
yes, good idea. Some simple endpoints to get/put new validators could be useful in case someone will want to automate/integrate. Should we consider adding swagger as well?

**Comment by jchen86** [comment]
I'm not sure about implementing this in the same project tbh - I certainly see the value of it, but we could potentially build this in a separate project and reuse code from here. I think keeping the feature list of staker-notifer to a minimal is really important, so it doesn't go down due to random reasons or unhandled API errors.
